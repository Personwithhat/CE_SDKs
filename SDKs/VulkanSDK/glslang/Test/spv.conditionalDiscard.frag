version https://git-lfs.github.com/spec/v1
oid sha256:c869d33196e2c1834c19ef84d0b63034d93e7235d1613373eb245ae6781aba7b
size 185
