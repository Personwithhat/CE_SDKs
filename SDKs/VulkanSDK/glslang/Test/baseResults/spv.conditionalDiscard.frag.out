version https://git-lfs.github.com/spec/v1
oid sha256:39fe2fa6b07759872c75bb3e35d032e8cf7515d9b6812bc2f119156c6f7c3495
size 2748
